src/file-2.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:46651/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:46651/project/issues?id=sample&pullRequest=6940&issues=AWlY8eebI7VaXCdnurPs&open=AWlY8eebI7VaXCdnurPs)

[ISSUE_KEY]: AWlY8eebI7VaXCdnurPs